Intersect.Client/Entities/Player.cs

```diff
@@ -1415,16 +1415,22 @@ public void AutoTarget()
                 return;
             }
 
-            if (mlastTargetList.ContainsKey(currentEntity))
+            Globals.Entities.TryGetValue(currentEntity.Id, out var targetedEntity);
+            if(targetedEntity == default)
```

Comment:
Just turn this into `!Globals.Entities.TryGetValue(...)` -- TryGetValue returns false if the out variable is default